src/settings/settingTab.ts

```diff
@@ -49,6 +49,10 @@ export class SettingTab extends PluginSettingTab {
 				"The target language can be selected by suggestion modal. The translation will be appended to the selection."
 			);
 
+        new Setting(containerEl)
+            .setName("Translate selection: Automatically append selection")
```

Comment:
It is already respected as long as you select the right command

For example: https://github.com/friebetill/obsidian-deepl/pull/12/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR38-R39